components/Service/ServicePage.js

```diff
@@ -0,0 +1,172 @@
+                        </div>
+                        <div className="intro-content">
+                            <i className="fas fa-phone icons-status" />
+                            <hr />
+                            <span className="services-content-text context3">
+                                <p className="intro-text1">
+                                   Emergency Contact Personel
+                                </p>
+                                <p className="intro-text2">{INTRO_TEXT2}</p>
+                            </span>
+                        </div>
+                        <div className="intro-content">
+                            <i className="fas fa-first-aid icons-status" />
+                            <hr />
+                            <span className="services-content-text context4">
+                                <p className="intro-text1">
+                                   Quick First Aid Solutions
+                                </p>
+                                <p className="intro-text2">{INTRO_TEXT2}</p>
+                            </span>
+                        </div>
+                    </div>
+                    <div className="content-wrapper services-wrapper intro-wrap">
+                        <div className="intro-content">
+                            <i className="fas fa-hospital icons-status hos" />
+                            <hr />
+                            <span className="services-content-text context2">
+                                <p className="intro-text1">
+                                   List of Hospitals Closest to You
+                                </p>
+                                <p className="intro-text2">{INTRO_TEXT2}</p>
+                            </span>
+                        </div>
+                        <div className="intro-content">
+                            <i className="fas fa-tv icons-status" />
+                            <hr />
+                            <span className="services-content-text context1">
+                                <p className="intro-text1">
+                                Medicial Screenings
+                                </p>
+                                <p className="intro-text2">{INTRO_TEXT2}</p>
+                            </span>
+                        </div>
+                        <div className="intro-content">
+                            <i className="fas fa-ambulance icons-status" />
+                            <hr />
+                            <span className="services-content-text context1">
+                                <p className="intro-text1">
+                                   Emergency Ambulance
+                                </p>
+                                <p className="intro-text2">{INTRO_TEXT2}</p>
+                            </span>
+                        </div>
+                    </div>
+                </div>
+            </div>
```

Comment:
map over all of this content

Reply:
Will do that once I start connecting it to the api directly

Reply:
Do that now @Merdoth, practice writing cleaner up.